src/client/visualizations/heat-map/heatmap-labels.tsx

```diff
@@ -16,41 +16,52 @@
  */
 
 import * as React from "react";
+import { noop } from "../../../common/utils/functional/functional";
+import { classNames } from "../../utils/dom/dom";
+import "./heatmap-labels.scss";
 
 interface HeatmapLabelsProps {
   labels: string[];
   orientation: "top" | "left";
   hoveredLabel: number;
+  highlightedLabel: number;
   labelSize?: number;
+
   onMaxLabelSize?(maxLabelSize: number): void;
 }
 
 export class HeatmapLabels extends React.Component<HeatmapLabelsProps> {
-  private container: HTMLDivElement | null = null;
+  private container = React.createRef<HTMLDivElement>();
 
   componentDidMount() {
-    if (this.container === null) {
+    if (this.container.current === null) {
       return;
     }
 
-    const {
-      onMaxLabelSize = () => {}
-    } = this.props;
+    const { onMaxLabelSize = noop } = this.props;
 
-    const maxWidth = Array.from(this.container.querySelectorAll(".heatmap-label"))
+    const maxWidth = Array.from(this.container.current.querySelectorAll(".heatmap-label"))
```

Comment:
Move `heatmap-label` to a constant, you have to synchronize this value with JSX.